src/modules/git/index.js

```diff
@@ -19,19 +19,25 @@ export default function gLog(path, doneCB, queryParameter, fileExtension) {
   // git command:
   // git log --merges --numstat -m --first-parent master --pretty=%cD --grep=bugfix/
 
-  const prefix = '*.';
-  if (fileExtension === '') {
-    fileExtension = '*';
+  if (fileExtentionExclusion === '') {
+    const prefix = '*.';
+    if (fileExtension === '') {
+      fileExtension = '*';
+    }
+    const fileExtensionArray = fileExtension.split(',');
+    const prefixString = prefix.concat(fileExtensionArray.join(',*.'));
+    const extensions = prefixString.split(',');
+    cmdArgs = cmdArgs.concat(extensions);
+  } else {
+    const prefix = '';
+    const prefix2 = prefix.concat('--,', '.,', ':^*.')
+    const fileExtentionExclusionArray = fileExtentionExclusion.split(',');
+    const prefixString = prefix2.concat(fileExtentionExclusionArray.join(',:^*.'));
+    const exclusions = prefixString.split(',');
```

Comment:
This is unnecessary. You are changing types constantly. From string to array to string to array :) should be possible with only one or two arrays